policies.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: blue_page
+permalink: /info/
```

Comment:
This is located under /info/. Any objection to renaming the file to info.md as well? (Or change the url to policies) to keep it consistent?